libretro-common/string/stdstring.c

```diff
@@ -26,6 +26,7 @@
 
 #include <string/stdstring.h>
 #include <encodings/utf.h>
+#include <stdio.h>
```

Comment:
Once `append_multi_disc_suffix()` moves into the database scanner .c files, stdio.h will no longer be required in this file.